docs/production-deployment/cloud/worker-health.mdx

```diff
@@ -287,13 +289,12 @@ poll_success_rate query
 **How to detect misconfigured Workers.**
```

Comment:
This is that example I was referring to above. It's really inconsistent where the bolded questions happen and where they don't right now. We should adopt and enforce one or the other.